src/amo/types/addons.js

```diff
@@ -108,7 +108,7 @@ export type GroupedRatingsType = {|
 export type ExternalAddonType = {|
   authors?: Array<AddonAuthorType>,
   average_daily_users: number,
-  categories?: Object | Array<string>,
+  categories?: Array<string>,
```

Comment:
Is there any way to properly validate this type?

I guess right now the only way would be to reference the schema for the addon type in our api swagger?

Outside of scope for this PR but I do think swagger supports type generation.. even api generation iirc.

Reply:
Current philosophy on our types in addons-frontend the team agreed on a while ago is that we maintain the flow types best we can but don't try to invest in more, because the costs outweigh the benefits. We also concluded back then that if we were to spend more time on it, we should switch to TypeScript first (which we looked into, but it was a significant effort)